test/screens/stops/stop_test.exs

```diff
@@ -42,7 +42,8 @@ defmodule Screens.Stops.StopTest do
         fn "stops",
            %{
              "filter[id]" => "s1,p1,p2,c3",
-             "include" => "child_stops,parent_station.child_stops"
+             "include" =>
+               "child_stops.connecting_stops,parent_station.child_stops,parent_station.connecting_stops,connecting_stops,parent_station"
```

Comment:
Although this updates the value we expect the fetch function to pass through to the API, it doesn't add any test coverage for the new parsing of `connecting_stops` (and this is not covered by any other tests since those will bypass this code entirely by injecting the fetch function). I think we should either augment the fake API response to include some connecting stops and assert that those too are parsed correctly, or add a new test case which does the same.

Reply:
good point, adding tests for this now!

Reply:
updated this test to process connecting_stops!